server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/CassandraMailQueueBrowser.java

```diff
@@ -131,6 +131,14 @@ public CassandraMailQueueItemView next() {
             .flatMapSequential(slice -> browseSlice(queueName, slice));
     }
 
+    Flux<EnqueuedItemWithSlicingContext> browseReferences(MailQueueName queueName, Instant start, Instant end) {
+        return allSlicesStartingAt(start)
+            .doOnNext(n -> System.out.println("     --- " + n + " slice looked up"))
```

Comment:
println?

Reply:
@Arsnael can you fix it up?

Reply:
I fixed it, but I do have a concern as I had my hands directly in the code. This newly created method does not seem to be used anywhere. So did you forget to switch to it from pieces of code using the other `browseReferences` method? Or is this useless and should be removed?

Reply:
cc @chibenwa

Reply:
True I moved away from it.

We can remove it.

Rationnals: as items prior browse start are all marked as deleted, then browse is not returning them. I had to use another strategy.

Please @Arsnael can you remove it?